dts/arm/microchip/mec172xnsz.dtsi

```diff
@@ -874,12 +874,17 @@
 		};
 		espi0: espi@400f3400 {
 			compatible = "microchip,xec-espi-v2";
+			/* reg tuple contains one 32-bit address cell and one
+			 * 32-bit length(size) cell.
+			 */
+			#address-cells = <1>;
+			#size-cells = <1>;
 			reg = < 0x400f3400 0x400
 				0x400f3800 0x400
 				0x400f9c00 0x400>;
 			reg-names = "io", "mem", "vw";
 			interrupts = <103 3>, <104 3>, <105 3>, <106 3>,
-				     <107 3>, <108 3>, <109 3>, <110 3>,
+				     <107 3>, <108 3>, <109 3>, <110 2>,
```

Comment:
Why is this being changed here? It looks like it might be a separate bug fix. The commit log is not clear about why these devicetree changes are needed in this patch in general.

Reply:
we are making espi reset interrupt a higher priority. Will add this information in the commit.

Reply:
> Will add this information in the commit.

The updated commit log says:

> espi reset interrupt is made a higer priority in MEC172x devicetree.

This is an explanation of **what** changes are being made, which I can see from the diff.

This is not an explanation of **why** the changes are being made. Please note that I wrote:

> The commit log is not clear about why these devicetree changes are needed in this patch in general.

Reply:
add explanation of why in the commit log